ab: add POST request example (#3805)
tldr-pages/tldr: pages/common/ab.md

- Set the maximum number of seconds to spend for benchmarking:

`ab -t {{60}} {{url}}`

> - Execute 100 HTTP POST requests to a given URL, using JSON payload stored in data.json file:

Reviewer: ```suggestion
- Execute 100 HTTP POST requests to a given URL, using a JSON payload from a file:
```
Could this line be reworded?
